inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/page/workload/DynamicWorkloadManagementPage.java

```diff
@@ -239,11 +235,9 @@ private void actionShowInfoDialog(AjaxRequestTarget aTarget, IModel<SourceDocume
         if (aProjectParam == null || aProjectParam.isEmpty()) {
```

Comment:
We do not use separator comments like `--- Helper methods ---`. Such methods are private - that's sufficient.